src/utils/error/error_utils_test.go

```diff
@@ -0,0 +1,36 @@
+package error
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+const (
+	errorMessage = "error message"
+)
+
+func TestNewInternalServerError(t *testing.T) {
+	err := NewInternalServerError(errorMessage)
+	assert.NotNil(t, err)
+	assert.EqualValues(t, http.StatusInternalServerError, err.Code())
+	assert.EqualValues(t, errorMessage, err.Message())
+	assert.EqualValues(t, fmt.Sprintf("message: %s - code: %d", errorMessage, 500), err.Error())
```

Comment:
You already imported the `"net/http"`, use `http.StatusInternalServerError` instead of the magic number
Do the same for the next ones.